invenio_edugain/ingest.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2025 Graz University of Technology.
+#
+# invenio-edugain is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Module for importing idp-data."""
+
+import dataclasses
```

Comment:
please try to use `from dataclasses import dataclass` statements

Reply:
Done.